service-statistics-unsd/src/main/java/fi/nls/oskari/control/statistics/plugins/unsd/parser/RegionMapper.java

```diff
@@ -0,0 +1,51 @@
+package fi.nls.oskari.control.statistics.plugins.unsd.parser;
```

Comment:
IMHO this class should be in the same package as `CountryRegion`

Reply:
I wouldn't even oppose removing `parser` and `requests` packages altogether especially since they only contain three classes in total.